CMake/AbseilHelpers.cmake

```diff
@@ -49,6 +49,8 @@ function(absl_library)
     PUBLIC ${ABSL_COMMON_INCLUDE_DIRS} ${ABSL_LIB_PUBLIC_INCLUDE_DIRS}
     PRIVATE ${ABSL_LIB_PRIVATE_INCLUDE_DIRS}
   )
+  # Add all Abseil targets to a a folder in the IDE for organization.
+  set_property(TARGET ${_NAME} PROPERTY FOLDER Abseil)
```

Comment:
Nit: Please set "Abseil" in a variable like ABSL_IDE_FOLDER or some other descriptive name instead of using the string directly.